vnext/Microsoft.ReactNative/Fabric/Composition/CompositionEventHandler.cpp

```diff
@@ -12,14 +12,15 @@
 #include <Views/ShadowNodeBase.h>
 #include <windows.h>
 #include <windowsx.h>
+#include <winrt/Windows.UI.Input.h>
 #include "Composition.Input.h"
 #include "CompositionRootView.h"
 #include "CompositionViewComponentView.h"
 #include "RootComponentView.h"
 
 namespace Microsoft::ReactNative {
 
-const PointerId MOUSE_POINTER_ID = 1; // TODO ensure this is something that does not conflict with pointer point IDs.
```

Comment:
The value's the same, is this no longer an issue?

Reply:
I verified that mouse is always 1.